.plop/templates/reactComponentTypes.ts.hbs

```diff
@@ -0,0 +1 @@
+export interface Props {}
```

Comment:
I would prefer to name it IProps or something like that, when i looked in reactComponent.tsx.hbs above I wasnt sure if Props is a type or interface

Reply:
👍